diff -p -up licq-1.3.7/src/licq.conf.h.licq_conf licq-1.3.7/src/licq.conf.h
--- licq-1.3.7/src/licq.conf.h.licq_conf	2008-06-14 04:08:14.000000000 +0200
+++ licq-1.3.7/src/licq.conf.h	2009-11-24 21:47:46.000000000 +0100
@@ -35,7 +35,7 @@ static const char LICQ_CONF[] =
 "\n"
 "[onevent]\n"
 "Enable = 1\n"
-"Command = play\n"
+"Command = soundwrapper play\n"
 "\n"
 "[groups]\n"
 "NumOfGroups = 0\n"
