diff -p -up licq-1.3.7/plugins/rms/src/rms.cpp.strfmt licq-1.3.7/plugins/rms/src/rms.cpp
--- licq-1.3.7/plugins/rms/src/rms.cpp.strfmt	2009-11-24 22:38:08.000000000 +0100
+++ licq-1.3.7/plugins/rms/src/rms.cpp	2009-11-24 22:47:57.000000000 +0100
@@ -1416,7 +1416,7 @@ int CRMSClient::Process_VIEW()
     strcat(szEventHeader, "\n\0");
 
     // Write out the event header
-    fprintf(fs, szEventHeader);
+    fprintf(fs, "%s",  szEventHeader);
 
     // Timestamp
     char szTimestamp[39];
@@ -1427,7 +1427,7 @@ int CRMSClient::Process_VIEW()
     sprintf(szTimestamp, "%d Sent At ", CODE_VIEWxTIME);
     strncat(szTimestamp, szTime, 25);
     strcat(szTimestamp, "\n\0");
-    fprintf(fs, szTimestamp);
+    fprintf(fs, "%s", szTimestamp);
 
     // Message
     fprintf(fs, "%d Message Start\n", CODE_VIEWxTEXTxSTART);
diff -p -up licq-1.3.7/src/licq.cpp.strfmt licq-1.3.7/src/licq.cpp
--- licq-1.3.7/src/licq.cpp.strfmt	2009-10-08 21:03:20.000000000 +0200
+++ licq-1.3.7/src/licq.cpp	2009-11-24 22:35:54.000000000 +0100
@@ -155,7 +155,7 @@ void ssl_info_callback(SSL *s, int where
 extern "C" void DisplayFatalError(const char* error, int useLicqLog)
 {
   if (useLicqLog)
-    gLog.Error(error);
+    gLog.Error("%s", error);
   else
     fprintf(stderr, "\n%s\n", error);
 
